app/models/degree_period.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: degree_periods
+#
+#  id         :integer          not null, primary key
+#  grad_month :string
+#  grad_year  :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class DegreePeriod < ApplicationRecord
+  has_one :accession
+
+  validates :grad_year, format: { with: /\A(19|20)\d{2}\z/i,
+                                  message: 'must be between 1900 and 2099' }
+
+  VALID_GRAD_MONTHS = %w[May June September February].freeze
```

Comment:
I consciously accepted the duplication, since DRYing it would effectively mean the beginning of the Big Refactor and may be confusing until we rip out the rest of the date logic from Thesis.

Reply:
Yeah that's fine by me. This is totally minor and will go away if we ever refactor in the way we probably should. 👍🏻